packages/mobile/src/screens/user-list-screen/TopSupportersScreen.tsx

```diff
@@ -31,18 +37,27 @@ const useStyles = makeStyles(({ spacing }) => ({
   },
   titleName: {
     maxWidth: 120
+  },
```

Comment:
nit: this could go in a `css` prop on the `Paper`?

Reply:
I don't think we have that css prop for native